src/ast/mod.rs

```diff
@@ -2980,6 +2980,32 @@ pub enum Statement {
         show_options: ShowStatementOptions,
     },
     /// ```sql
+    /// SHOW [ TERSE ] OBJECTS [ LIKE '<pattern>' ]
+    ///                        [ IN
+    ///                              {
+    ///                                ACCOUNT                                         |
+    ///    
+    ///                                DATABASE                                        |
+    ///                                DATABASE <database_name>                        |
+    ///
+    ///                                SCHEMA                                          |
+    ///                                SCHEMA <schema_name>                            |
+    ///                                <schema_name>
+    ///
+    ///                                APPLICATION <application_name>                  |
+    ///                                APPLICATION PACKAGE <application_package_name>  |
+    ///                              }
+    ///                        ]
+    ///                        [ STARTS WITH '<name_string>' ]
+    ///                        [ LIMIT <rows> [ FROM '<name_string>' ] ]
+    /// ```
```

Comment:
Thinking since this is quite verbose we can either leave it out, relying on the link to the documentation and a description or we can use a shorter summary of the syntax like an example sql statement?